src/Sarif/Autogenerated/Address.cs

```diff
@@ -42,7 +43,12 @@ public SarifNodeKind SarifNodeKind
         [DataMember(Name = "absoluteAddress", IsRequired = false, EmitDefaultValue = false)]
         [DefaultValue(-1)]
         [JsonProperty(DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate)]
-        public int AbsoluteAddress { get; set; }
+        public BigInteger AbsoluteAddress { get; set; }
+
+        public bool ShouldSerializeAbsoluteAddress()
+        {
+            return AbsoluteAddress >= 0;
+        }
```

Comment:
this is needed because DefaultValue does not work for BigInteger.

then this need 2 things,
1. in the constructor to set it, which we already have.
2. ShouldSerializeXXX to handle the serialization.

Reply:
Normally, this is placed in the core folder:
[https://github.com/microsoft/sarif-sdk/blob/41c12efef265094d9fb0dcf35eb890fc35079337/src/Sarif/Core/Run.cs#L194-L211](https://github.com/microsoft/sarif-sdk/blob/41c12efef265094d9fb0dcf35eb890fc35079337/src/Sarif/Core/Run.cs#L194-L211)

Reply:
Just notice this, thanks for the info!